stable/aws-for-fluent-bit/values.yaml

```diff
@@ -93,6 +92,30 @@ cloudWatch:
   # extraOutputs: |
   #   ...
 
+cloudWatchLogs:
+  enabled: true
+  match: "*"
+  region: "us-east-1"
+  logGroupName: "/aws/eks/fluentbit-cloudwatch-native/logs"
+  logGroupTemplate: /aws/eks/fluentbit-cloudwatch-native/workload/$kubernetes['namespace_name']
+  logStreamName:
+  logStreamPrefix: "fluentbit-"
+  logStreamTemplate: $kubernetes['pod_name'].$kubernetes['container_name']
```

Comment:
I think I am fine with having templating be default here but I think we should keep the same default/fallback log group name/prefix as the old chart?

`fluentbit-cloudwatch-native` => why is noting the plugin name important in the pod log group name? I wouldn't have ever put `fluentbit` in the old name personally.

Reply:
fair point. updated